packages/composites/token-allocation/src/adapter.ts

```diff
@@ -6,6 +6,8 @@ import { AdapterError, Requester, Validator } from '@chainlink/ea-bootstrap'
 import { BigNumber } from 'ethers'
```

Comment:
Could we get rid of this now and just use Decimal in its place?

Reply:
We agreed some time ago, BigNumber should be for integer operations, and Decimal for floating

Reply:
That works, pushing a README update so that we have this pattern documented, otherwise it will get lost.